Add note for streaming TraceQL results for GET and/or Tempo setups using intermediary proxies (#89251)
grafana/grafana: docs/sources/shared/datasources/tempo-editor-traceql.md

To use the TraceQL query editor in self-hosted Grafana 9.3.2 and older, you need to [enable the `traceqlEditor` feature toggle](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/).

> If trying to query a self-managed Grafana Tempo or Grafana Enterprise Traces database with a gateway (e.g., nginx) in front of it from your hosted Grafana, that gateway (e.g., nginx) must allow gRPC connections. If it does not, streaming will not work and queries will fail to return results.
> 
> If you cannot configure your gateway to allow gRPC, open a support escalation to request streaming query results be disabled in your hosted Grafana.
> 

Reviewer: We shouldn't use e.g.: https://developers.google.com/style/word-list#eg. I need to add a Vale rule for it
```suggestion
If you run a gateway (like nginx) in front of self-managed Grafana Tempo or Grafana Enterprise Traces database, and you want to query that database from Hosted Grafana, the gateway must allow gRPC connections.
If it doesn't, streaming won't work and queries won't return results.

If you can't configure your gateway to allow gRPC, open a support escalation to request streaming query results be disabled in your hosted Grafana.
```

To use the TraceQL query editor in self-hosted Grafana 9.3.2 and older, you need to [enable the `traceqlEditor` feature toggle](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/).

> To enable streaming on Grafana Cloud using an on-prem Grafana Enterprise Traces instance, the gateway (ngnix) should allow gRPC connections.

Author: ```suggestion
To enable streaming on Grafana Cloud using an on-prem Grafana Enterprise Traces instance, the gateway (nginx) should allow gRPC connections.
```
